packages/mux-player/src/index.ts

```diff
@@ -456,23 +485,20 @@ class MuxPlayerElement extends VideoApiElement {
 
   /**
    * Set the start time.
-   * @param  {number} val
```

Comment:
these aren't necessary because we have the types file?

Reply:
yes, most of the types are also inferred